code/js/content.js

```diff
@@ -289,7 +289,16 @@ function sanitize(text) {
 $(document).ready(function() {
   var $searchContainer = $('form#search_form, .site-search');
   var $q = $searchContainer.find('input[name="q"]');
-  var isRepository = $searchContainer.hasClass('repo-scope');
+  var $scopeBadge = $searchContainer.find('.scope-badge');
+  var $scopedSearch = $q.closest('.scoped-search');
+
+  function isRepository() {
+    return $scopeBadge.is(':visible') && $scopeBadge.text() === 'This repository';
+  }
+
+  function isOrganization() {
+    return $scopeBadge.is(':visible') && $scopeBadge.text() === 'This organization';
+  }
 
```

Comment:
That's unfortunately the best I could think of for now to detect in which scope we are.